src/test/java/org/owasp/html/ElidedCharactersTest.java

```diff
@@ -0,0 +1,143 @@
+package org.owasp.html;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import junit.framework.TestCase;
+import org.junit.Test;
+
+/**
+ * Some characters should not appear in HTML documents, present risks for log-file injection, or are otherwise discouraged from sanitized HTML. This set of
+ * unit tests verifies that the inclusion of such characters does not allow dangerous code to slip through.
+ * <p>
+ * There are two requirements:
+ * <p>
+ * 1) The Encoding.encodeRcdataOnto method should remove discouraged characters.
+ * 2) Sanitized HTML should not change
+ *
+ * @author Simon Greatrix on 25/01/2021.
+ */
+public class ElidedCharactersTest extends TestCase {
```

Comment:
I'll look at this PR in more detail shortly.  In the meantime, these tests seem to be failing on Travis.  Does that differ from what you see running locally?